src/server/ssrServer.js

```diff
@@ -131,7 +134,7 @@ export async function createApp(opts = {}) {
       });
     } else {
       instance.post('/_/report/security/csp-violation', (request, reply) => {
-        const violation = request.body ? JSON.stringify(request.body, null, 2) : 'No data received!';
+        const violation = request.body ? request.body : 'No data received!';
```

Comment:
Why does development require a JSON.parse, but prod does not?

Reply:
the content parser already parses as a string, there is no need to call `JSON.parse` immediately followed by a `JSON.stringify`for production. In development the string is being formatted to follow a different shape. https://github.com/americanexpress/one-app/pull/1038/files#diff-24bcfbf7958e3957b02a997dace0e41e8a46eb5501b8157d05a4dc4cecb75576L119-L128